react-props/react-props-and-expressions-quiz-notes.md

```diff
@@ -0,0 +1,49 @@
+# react-props-and-expressions-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What are props in React?
+
+the means of passing arguments to React components when they are called
+
+- How do you declare and access props in a component?
+
+by using Type
+
+- How do you pass props to a component?
+
+adding them to teh JSX
```

Comment:
Add what to what JSX? What syntax needs to happen to pass a prop to a component?